index.css

```diff
@@ -0,0 +1,14 @@
+#image {
+    display: block;
+      
+    max-width: 100%;
+    
+    margin-left: auto;
+    
+    margin-right: auto;
+    
+}
+
+.align{
```

Comment:
Please add a space after every class. This is a common linter errors that should be avoided. Linters help make our code easier to read along with making it seem more professional.